test/minikube/minikube_insights_test.go

```diff
@@ -54,28 +54,21 @@ func startAndScaleYCSB(t *testing.T, namespaceName string, options *helm.Options
 
 func checkMetricPresent(t *testing.T, namespace string, influxPodName string, influxDatabase string,
 	measurement string, database string, host string, metric string) bool {
-	queryString := fmt.Sprintf("select count(%s) from \"%s\" where host = '%s'", metric, measurement, host)
-	dbTagName := "db"
-	if influxDatabase == "nuodb_internal" {
-		// DB tag name is different in nuodb_internal database
-		dbTagName = "dbname"
-	}
-	if measurement != "nuodb_thread" {
-		// There is no db tag for nuodb_thread measurement
-		queryString = fmt.Sprintf("%s and %s = '%s'", queryString, dbTagName, database)
-	}
-	output, err := ExcuteInfluxDBQueryE(t, namespace, influxPodName, queryString, "-database", influxDatabase, "-format", "csv")
+	// queryString := fmt.Sprintf("select count(%s) from \"%s\" where host = '%s'", metric, measurement, host)
+	// dbTagName := "db"
+	query := fmt.Sprintf("from(bucket: \"%s\")\n |> range(start: -5m)\n |> filter(fn : (r) => r[\"_measurement\"] == \"%s\" and r[\"_field\"] == \"%s\" and r[\"host\"] == \"%s\") |> keep(columns: [\"_value\"]) |> count()", influxDatabase, measurement, metric, host)
```

Comment:
This is a very long string that we're building. Can you include a comment saying what it does?